frontend/schemas/project.ts

```diff
@@ -206,7 +208,25 @@ export default (page: number) => {
     }),
     object().shape({
       description: string().max(600, maxTextLength).required(messages.description),
-      relevant_links: string().nullable(),
+      relevant_links: string().max(600, maxTextLength).nullable(),
+      positive_financial_returns: string().max(600, maxTextLength).nullable(),
+      last_year_sales_revenue: number()
+        .positive()
+        .test(
+          'is-decimal',
+          'The amount should be a decimal with maximum two digits after comma',
+          (val: any) => {
+            if (val != undefined) {
+              return patternTwoDigisAfterComma.test(val);
+            }
+            return true;
+          }
+        )
+        .min(0)
+        .max(999999999.99)
+        .nullable(),
```

Comment:
This input should be optional, but if I have a value I can't erase the form without a error

<img width="865" alt="Screenshot 2023-11-28 at 11 38 56" src="https://github.com/Vizzuality/heco-invest/assets/48164343/12a41e0e-df34-4af5-b2a7-a9a96a7b5749">